CPAC/nuisance/utils/compcor.py

```diff
@@ -92,17 +92,37 @@ def cosine_filter(
 ):
     """
     `cosine_filter` adapted from Nipype.
-
     https://github.com/nipy/nipype/blob/d353f0d/nipype/algorithms/confounds.py#L1086-L1107
-
-    Parameters
-    ----------
-    input_image_path : string
-            Bold image to be filtered.
+    
+    Parameters:
+    -----------
```

Comment:
Maybe I'm missing something, but why add colons to [the section headers](https://numpydoc.readthedocs.io/en/latest/format.html)?

```suggestion

    Parameters
    ----------
```